examples/mnist_stages/config.yml

```diff
@@ -51,8 +51,19 @@ stages:
       # default kwargs for `runner.get_loaders`:
       batch_size: 32
       num_workers: 1
-      # kwargs for `runner.get_datasets`:
-      num_samples_per_class: 320
+
+      datasets:
+        train:
+          # _target_: MNIST
```

Comment:
why we have not used `_target_`?

Reply:
why could not you fix them in this one? you already using `registry.get` for datasets

Reply:
moreover, you are using "registry"-like defenition under Hydra API

Reply:
fixed